### PR TITLE
thorvg: New 0.13.2 version and several improvements

### PR DESCRIPTION
**Pull request by franramirez688**
* Added new patch version: `0.13.2`
* Added option `with_simd` (legacy option `vector` renamed to `simd` upstream)
* Added explicit requirements.

cc @toge

### CODE REVIEW

**Review by toge — APPROVED** [review]
Thanks!

File: recipes/thorvg/all/conanfile.py [hunk 1]
```diff
@@ -77,6 +80,8 @@ def _compilers_minimum_version(self):
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
+        if Version(self.version) > "0.13.0":
```
Comment (RubenRBS) [inline recipes/thorvg/all/conanfile.py:83]: But does the `with_simd` still have an effect in old versions? What happens if their values differs in versions where both exist?

**Review by RubenRBS — COMMENTED** [review]
One question about the handling of the new option, but looks good overall. Thanks for adding option descriptions! :)

File: recipes/thorvg/all/conanfile.py (hunk 1 above)
Comment (franramirez688) [reply recipes/thorvg/all/conanfile.py:83]: The thing is that it does not exist in the old versions. Perhaps, it is not clear in the changelog 😓
In summary:

* Versions < 0.13.0 -> `vector`
* Versions >= 0.13.1 -> ~`vector`~ `simd`

Comment (franramirez688) [reply recipes/thorvg/all/conanfile.py:83]: See here https://github.com/thorvg/thorvg/compare/v0.13.0...v0.13.1#diff-76ea9078099f2ee8cc0e760a874dd153f225ba16ad10375c24ab6cd53518f3c4L24

**Review by RubenRBS — DISMISSED** [review]

File: recipes/thorvg/all/conanfile.py [hunk 2]
```diff
@@ -54,7 +56,8 @@ class ThorvgConan(ConanFile):
         "loaders": "Enable File Loaders in thorvg",
```
Comment (valgur) [inline recipes/thorvg/all/conanfile.py:56]: The option names in `options_description` should be adjusted to match their names in `options` and `default_options`.

File: recipes/thorvg/all/conanfile.py [hunk 3]
```diff
@@ -33,7 +33,8 @@ class ThorvgConan(ConanFile):
         "with_bindings": [False, 'capi', 'wasm_beta'],
         "with_tools": [False, 'svg2tvg', 'svg2png', 'lottie2gif', 'all'],
         "with_threads": [True, False],
-        "with_vector": [True, False],
+        "with_vector": [True, False],  # removed in thorvg 0.13.1. Renamed to simd
+        "with_simd": [True, False],  # legacy with_vector
```
Comment (valgur) [inline recipes/thorvg/all/conanfile.py:37]: I would mark the legacy option as deprecated (set the default value to `"deprecated"` and log a warning if it is being set).

Comment (franramirez688) [reply recipes/thorvg/all/conanfile.py:37]: But this case is different. This case is renamed so it is not a "deprecated" issue at all. I think that putting "deprecated" is a bit confusing as it does not exist anymore.

**Review by RubenRBS — APPROVED** [review]

**Review by toge — APPROVED** [review]

**Pull request merged**